e2e/pages/adf/contentServicesPage.ts

```diff
@@ -61,16 +61,24 @@ export class ContentServicesPage {
     copyButton = element(by.css('button[data-automation-id="content-node-selector-actions-choose"]'));
     searchInputElement = element(by.css('input[data-automation-id="content-node-selector-search-input"]'));
     shareNodeButton = element(by.cssContainingText('mat-icon', ' share '));
+    nameColumn = by.css('div[class*="datatable-body"] div[class*="adf-data-table-cell"][title="Display name"]');
+    nameColumnHeader = by.css('div[data-automation-id="auto_id_name"]');
```

Comment:
Same comment as above, we should just provide "auto_id_name" or something else from this Page Object and datatable should know how to create the columnHeader locator with this information